src/index.jsx

```diff
@@ -2,7 +2,6 @@ import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 import { Theme } from '@radix-ui/themes';
 import { App } from './App';
-import { Theme } from '@radix-ui/themes';
 
```

Comment:
I actually deleted this line and pushed it to main earlier, as I had seen a comment regarding this on Slack.